cms/templates/js/course-outline.underscore

```diff
@@ -78,12 +78,14 @@ if (xblockInfo.get('graded')) {
                         </a>
                     </li>
                 <% } %>
-                <li class="action-item action-delete">
-                    <a href="#" data-tooltip="<%= gettext('Delete') %>" class="delete-button action-button">
-                        <i class="icon fa fa-trash-o"></i>
-                        <span class="sr action-button-text"><%= gettext('Delete') %></span>
-                    </a>
-                </li>
+                <% if (!(xblockInfo.isEntranceExam())) { %>
```

Comment:
I'd rather introduce a new method such as ```isReadOnly``` or ```canBeDeleted```. It doesn't seem right that entrance exams have to be handled specially in the template.